File: 0001_0100/0041/0041.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n = int(f.readline())
    p = [0] * 201
    for x in f.read().split():
        p[int(x) + 100] += 1
    for i in range(201):
        if p[i]:
            q.write(' '.join([str(i - 100)] * p[i]) + ' ')

File: 0001_0100/0041/0041.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, i, k;
	int p[201] = {0};

	fscanf(f, "%d", &n);
	for (i = 0; i < n; ++i)
	{
		fscanf(f, "%d ", &k);
		++p[k + 100];
	}
	for (i = 0; i < 201; ++i)
		while(p[i]--)
			fprintf(q, "%d ", i - 100);
	return 0;
}
